addons/account/models/account_lock_exception.py

```diff
@@ -277,52 +277,54 @@ def _get_active_exceptions_domain(self, company, soft_lock_date_fields):
-            domain.append(('date', '<=', self.end_datetime))
+            common_message_domain.append(('date', '<=', self.end_datetime))
 
-        # Add a restriction on the accounting date to avoid unnecessary entries
+        # Add restrictions on the accounting date to avoid unnecessary entries
         min_date = self.lock_date
         max_date = self.company_lock_date
         move_date_domain = []
         tracking_old_datetime_domain = []
         tracking_new_datetime_domain = []
         if min_date:
-            move_date_domain.append([('account_audit_log_move_id.date', '>=', min_date)])
+            move_date_domain.append([('date', '>=', min_date)])
             tracking_old_datetime_domain.append([('tracking_value_ids.old_value_datetime', '>=', min_date)])
             tracking_new_datetime_domain.append([('tracking_value_ids.new_value_datetime', '>=', min_date)])
         if max_date:
-            move_date_domain.append([('account_audit_log_move_id.date', '<=', max_date)])
+            move_date_domain.append([('date', '<=', max_date)])
             tracking_old_datetime_domain.append([('tracking_value_ids.old_value_datetime', '<=', max_date)])
             tracking_new_datetime_domain.append([('tracking_value_ids.new_value_datetime', '<=', max_date)])
-        domain.extend([
+
+        return [
+            ('company_id', 'child_of', self.company_id.id),
             '|',
-                *expression.AND(move_date_domain),
-                '&',
+                # The date was changed from or to a value inside the excepted period
+                ('audit_trail_message_ids', 'any', [
+                    *common_message_domain,
                     ('tracking_value_ids.field_id', '=', self.env['ir.model.fields']._get('account.move', 'date').id),
                     '|',
                         *expression.AND(tracking_old_datetime_domain),
                         *expression.AND(tracking_new_datetime_domain),
-        ])
-
-        return domain
+                ]),
+                # The date of the move is inside the excepted period and sth. was changed on the move
+                '&',
+                    *expression.AND(move_date_domain),
+                    ('audit_trail_message_ids', 'any', [
+                        *common_message_domain,
+                    ]),
+        ]
 
     def action_show_audit_trail_during_exception(self):
         self.ensure_one()
         return {
-            'name': _("Audit Trail during the Exception"),
+            'name': _("Journal Items"),
             'type': 'ir.actions.act_window',
-            'res_model': 'mail.message',
-            'views': [(self.env.ref('account.view_message_tree_audit_log').id, 'list'), (False, 'form')],
-            'search_view_id': [self.env.ref('account.view_message_tree_audit_log_search').id],
-            'domain': self._get_audit_trail_during_exception_domain(),
-        }
+            'res_model': 'account.move.line',
```

Comment:
Why do we show the lines if we have information only on the moves?

Reply:
https://github.com/odoo/odoo/pull/181360#issuecomment-2373650195
If I remember correctly: This way it is easier to see what impact the changes may have.
And it is still easy to access individual moves via the line or just group by moves to get an overview of moves
@tsb-odoo